js/compiled/tna-base.js

```diff
@@ -85,6 +85,14 @@ $.fn.webtrends_click_handler = function () {
     })
 };
 
+//Toggles class of minus when clicked.
+$.fn.toggleMinusClass = function (){
```

Comment:
@punalsc - this doesn't seem to have been implemented correctly. As a plugin it should act upon the jQuery collection to which it is applied. The way it has been implemented the function body makes no reference to the jQuery collection and it instead defines a selector within the function body.